server/server.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http')
+const port = 3000
+const requestHandler = (request, response) => {
+    console.log(request.url)
```

Comment:
Nice! Could you also output `request.method` and parse URL into parts like in the following example

~~~js
require('url').parse('/status?name=ryan')
Url {
  protocol: null,
  slashes: null,
  auth: null,
  host: null,
  port: null,
  hostname: null,
  hash: null,
  search: '?name=ryan',
  query: 'name=ryan',
  pathname: '/status',
  path: '/status?name=ryan',
  href: '/status?name=ryan' }
~~~